jbpm-form-modeler-core/jbpm-form-modeler-service/jbpm-form-modeler-ui/src/main/java/org/jbpm/formModeler/core/processing/impl/FormProcessorImpl.java

```diff
@@ -45,49 +45,63 @@
     private Logger log = LoggerFactory.getLogger(FormProcessor.class);
 
     // TODO: fix formulas
-    @Inject
     private FormulasCalculatorChangeProcessor formChangeProcessor;
 
-    @Inject
     private RangeProviderManager rangeProviderManager;
 
-    @Inject
     private DefaultFormulaProcessor defaultFormulaProcessor;
 
-    @Inject
     private FieldHandlersManager fieldHandlersManager;
 
-    @Inject
     private FormRenderContextManager formRenderContextManager;
 
-    @Inject
     private NamespaceManager namespaceManager;
 
-    @Inject
```

Comment:
@pefernan just trying to understand... why did you remove the @Inject annotations here?